content/en/docs/reference/glossary/mesh-federation.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Mesh Federation
+---
+A loosely-coupled set of autonomous [service meshes](#service-mesh) collectively providing the services
```

Comment:
An application probably cannot span multiple meshes (for security and isolation reasons), and that's definitely not the key principle behind a federation of meshes.

Here's a possible replacement first sentence:

A loosely-coupled set of [service meshes](#service-mesh) that supports inter-mesh communication.

BTW I saw our current doc says something about a mesh being a way to create a composite application, that's incorrect as well, hopefully we can fix that as part of updating the docs.

Reply:
I would say Vadim's example is exactly that, an application that requires some of its services (e.g., the payment card processing) to be isolated from the rest of the application. You could argue they are separate applications, but I would say these are two components of a single application, separated for regulatory compliance reasons.

Yes, that doc is to be replaced by Nathan and Rigs' Istio Topology PR, which is supposed to be ready for review any time now. So here we just need to agree on a correct definitions for the glossary terms.